p1pop.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link href="s1pop.css" type="text/css" rel="stylesheet">
```

Comment:
Dlaczego `s1pop.css` a nie `style.css` ? Dobra praktyka, używajmy przyjętych standardów w nazewnictwie :) Nazwa pliku `s1pop` nic mi nie mówi. A jakbym miał zgadywać to to że są to style do jakiegoś `popup`a ? :)